src/Integrations/Jira/SettingsJira.php

```diff
@@ -0,0 +1,393 @@
+		}
+
+		return true;
+	}
+
+	/**
+	 * Get Form settings data array
+	 *
+	 * @param string $formId Form Id.
+	 *
+	 * @return array<int, array<string, mixed>>
+	 */
+	public function getSettingsData(string $formId): array
+	{
+		// Bailout if feature is not active.
+		if (!$this->isSettingsGlobalValid()) {
+			return $this->getNoActiveFeatureOutput();
+		}
+
+		$selectedProject = $this->getSettingsValue(self::SETTINGS_JIRA_PROJECT_KEY, $formId);
+		$selectedIssueType = $this->getSettingsValue(self::SETTINGS_JIRA_ISSUE_TYPE_KEY, $formId);
+
+		return [
+			$this->getIntroOutput(self::SETTINGS_TYPE_KEY),
+			[
+				'component' => 'tabs',
+				'tabsContent' => [
+					[
+						'component' => 'tab',
+						'tabLabel' => \__('Settings', 'eightshift-forms'),
+						'tabContent' => [
+							[
+								'component' => 'select',
+								'selectName' => $this->getSettingsName(self::SETTINGS_JIRA_PROJECT_KEY),
+								'selectFieldLabel' => \__('Select a project', 'eightshift-forms'),
+								'selectSingleSubmit' => true,
+								'selectContent' => \array_merge(
+									[
+										[
+											'component' => 'select-option',
+											'selectOptionLabel' => '',
+											'selectOptionValue' => '',
+										],
+									],
+									\array_map(
+										static function ($option) use ($selectedProject) {
+											return [
+												'component' => 'select-option',
+												'selectOptionLabel' => $option['title'],
+												'selectOptionValue' => $option['key'],
+												'selectOptionIsSelected' => $selectedProject === $option['key'],
+											];
+										},
+										$this->jiraClient->getProjects()
+									)
+								),
+							],
+							$selectedProject ? [
+								'component' => 'select',
+								'selectSingleSubmit' => true,
+								'selectName' => $this->getSettingsName(self::SETTINGS_JIRA_ISSUE_TYPE_KEY),
+								'selectFieldLabel' => \__('Select a issue type', 'eightshift-forms'),
+								'selectContent' => \array_merge(
+									[
+										[
+											'component' => 'select-option',
+											'selectOptionLabel' => '',
+											'selectOptionValue' => '',
+										],
+									],
+									\array_map(
+										static function ($option) use ($selectedIssueType) {
+											return [
+												'component' => 'select-option',
+												'selectOptionLabel' => $option['title'],
+												'selectOptionValue' => $option['id'],
+												'selectOptionIsSelected' => $selectedIssueType === $option['id'],
+											];
+										},
+										$this->jiraClient->getIssueType($selectedProject)
+									)
+								),
+							] : [],
+						],
+					],
+					$selectedIssueType ? [
+						'component' => 'tab',
+						'tabLabel' => \__('Options', 'eightshift-forms'),
+						'tabContent' => [
+							[
+								'component' => 'input',
+								'inputName' => $this->getSettingsName(self::SETTINGS_JIRA_TITLE_KEY),
+								'inputFieldLabel' => \__('Issue title', 'eightshift-forms'),
```

Comment:
```suggestion
								'inputFieldLabel' => \__('Title', 'eightshift-forms'),
```

Reply:
Issue title is mandatory because we have epic title later so I would like to leave this as is